.jshintrc

```diff
@@ -151,5 +152,5 @@
 	// Maximum number of statements per function.
 	"maxstatements": 50,
 	// Maximum cyclomatic complexity.
-	"maxcomplexity": 10
```

Comment:
not really a comment about this PR, but it'd be nice to move to eslint at some point IMO :)